packages/react/checkbox/src/Checkbox.tsx

```diff
@@ -57,10 +57,11 @@ const Checkbox = React.forwardRef((props, forwardedRef) => {
   } = props;
   const [button, setButton] = React.useState<HTMLButtonElement | null>(null);
   const composedRefs = useComposedRefs(forwardedRef, (node) => setButton(node));
-  const buttonSize = useSize(button);
```

Comment:
Moved this into `BubbleInput` because it's only needed if we render `BubbleInput`